### ISSUE TITLE
Make library tree shakeable

### ISSUE THREAD

**Issue by nfantone** [flybondi/ramda-land#16]
As [reported by BundlePhobia](https://bundlephobia.com/result?p=@flybondi/ramda-land@4.0.1), `ramda-land` is not tree-shakeable.

Useful resources:

- [`ramda-adjunct@2.28.0`](https://bundlephobia.com/result?p=ramda-adjunct@2.28.0) BundlePhobia report.
- Related [`package.json` fields](https://github.com/char0n/ramda-adjunct/blob/master/package.json#L60) to enable tree-shaking on Webpack.
- Official [Webpack docs on Tree Shaking](https://webpack.js.org/guides/tree-shaking/).

TLDR;

> [...] in order to take advantage of tree shaking, you must...
>
> - Use ES2015 module syntax (i.e. `import` and `export`).
> - Ensure no compilers transform your ES2015 module syntax into CommonJS modules.
> - Add a `"sideEffects"` property to your project's `package.json` file.
> - Use the `production` mode configuration option to enable various optimizations including minification and tree shaking.
